viral/multiple_sessions.py

```
@@ -7,12 +7,13 @@
 import sys
 import inspect
 from pydantic import ValidationError
+import pandas as pd
```

Ruff (0.12.2)

Redefinition of unused `pd` from line 5

Remove definition: `pd`

(F811)